src/components/calcite-input/calcite-input.e2e.ts

```diff
@@ -670,7 +670,7 @@ describe("calcite-input", () => {
 
   it("allows restricting input length", async () => {
     const page = await newE2EPage({
-      html: `<calcite-input minLength="2" maxLength="3"></calcite-input>`
+      html: `<calcite-input minLength="2" maxLength="3" value=""></calcite-input>`
```

Comment:
Is this needed for the test to pass? Or I should say, does the initial value affect the input length restriction?

Reply:
Yes, it is needed for the test to pass for some reason.  I have been struggling with getting that test to pass for days, and for some reason doing that makes it pass even though in all the browsers I tested this use case with it all worked as expected without having to set the initial value like that.

Reply:
Sounds like regression based on that. Do you know exactly at which point the test fails?

Reply:
It fails only when it is checking validity when the value is set to `1` with a `minlength` of 2.  It passes on the subsequent assertions.